docs/src/main/sphinx/overview/concepts.md

```diff
@@ -81,72 +81,126 @@ for task execution.
-of `hive.test_data.test` refers to the `test` table in the
-`test_data` schema in the `hive` catalog.
-
-Catalogs are defined in properties files stored in the Trino
-configuration directory.
+A Trino catalog is a collection of configuration properties to access a specific
+data source, including the required connector and any other details such as
+credentials and URL. Catalogs are defined in properties files stored in the
+Trino configuration directory. The name of the properties file determines the
+name of the catalog. For example, the properties file `etc/example.properties`
+results in a catalog name `example`.
+
+You can configure and use many catalogs, also using the same connector, to
+access different data sources. For example, if you have two data lakes, you can
+configure two catalogs in a single Trino cluster that both use the Hive
+connector, allowing you to query data from both clusters, even within the same
+SQL query. You can also use a Hive connector for one catalog to access a data
+lake, and use the Iceberg connector for another catalog to access the data
+lakehouse. Or you can configure different catalogs to access different
+PostgreSQL database. The combination of different catalog is determined by your
+needs to access different data sources only.
+
+A catalog contains one or more schemas, which in turn contain objects such as
+tables, views, or materialized views. When addressing an object like table in
+Trino, the fully-qualified table name is always rooted in a catalog. For
+example, a fully-qualified table name of `example.test_data.test` refers to the
+`test` table in the `test_data` schema in the `example` catalog.
 
 ### Schema
 
-Schemas are a way to organize tables. Together, a catalog and schema
-define a set of tables that can be queried. When accessing Hive or a
-relational database such as MySQL with Trino, a schema translates to
-the same concept in the target database. Other types of connectors may
-choose to organize tables into schemas in a way that makes sense for
-the underlying data source.
+Schemas are a way to organize tables. Together, a catalog and schema define a
+set of tables and other objects that can be queried. When accessing Hive or a
+relational database such as MySQL with Trino, a schema translates to the same
+concept in the target database. Other types of connectors may choose to organize
+tables into schemas in a way that makes sense for the underlying data source.
 
 ### Table
 
-A table is a set of unordered rows, which are organized into named columns
-with types. This is the same as in any relational database. The mapping
-from source data to tables is defined by the connector.
+A table is a set of unordered rows, which are organized into named columns with
+[types](/language/types). This is the same as in any relational database. The
+mapping from source data to tables is defined by the connector. Type mapping
+varies across connectors and is documented in the specific connector
```

Comment:
Type mapping from sou...by the connector, and varies across

Reply:
updated